common/buildings/14_branch_office_buildings.txt

```diff
@@ -419,8 +446,8 @@ building_wildcat_miners = {
 	potential = {
 		has_branch_office = yes
```

Comment:
I can't see pushbacks on the criminal syndicate buff we have done, did we revert that? I think that would be important for balance this patch.